### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by Mugiira-Kim**
In this assignment I was able to complete the following; 

- Add a Navigation bar with three content sections:
   1. Books list.
   2. A book form.
   3. Contact info.
When a user clicks on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Mugiira-Kim, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Navigation bar ✔
- No linter errors ✔

## Required Changes ♻️
- [ ] When any of the links are clicked, the main area should change to display only one of the three content sections. Let's fix that

https://user-images.githubusercontent.com/40503704/219867879-afdfa988-03e4-4a75-a201-c7a96abe9133.mov


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -1,56 +1,77 @@
...
-    this.addBook = this.addBook.bind(this);
-    this.removeBook = this.removeBook.bind(this);
-
-    this.addBookButton.addEventListener('click', this.addBook);
-    this.bookList.addEventListener('click', this.removeBook);
-
-    this.render();
-  }
+// eslint-disable-next-line no-undef
+const { DateTime } = luxon;
```
Comment (AmaduKamara) [inline script.js:2]: [ Optional Changes ]

- [ ] Let's try not to disable Linter checks in our codebase as Linter checks are there to help us write clean, correct, and professional code. Kindly solve linter errors and have your codebase professional and error-free from Linter cheks 👍🏽

File: site.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+// eslint-disable-next-line no-undef
+const { DateTime } = luxon;
```
Comment (AmaduKamara) [inline site.js:2]: [ Optional Changes ]

- [ ] Let's try not to disable Linter checks in our codebase as Linter checks are there to help us write clean, correct, and professional code. Kindly solve linter errors and have your codebase professional and error-free from Linter cheks 👍🏽

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @Mugiira-Kim,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ All three sections work fine as expected when the nav links are clicked 👍🏽
✅ Add and remove book features work fine 👍🏽

## Recommendations:

- [ ] What if you could make the List Books the default view instead of an empty view until one has to click a link on the navbar? It'll be good for someone to least see the list of books or the message "All Awesome Books" and then when the add book link is clicked, then a book is added, and so on 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**